ShopsTable.API/Models/Item.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ShopsTable.API.Models
+{
+    public class Item
+    {
+        [Key]
+        public Guid Id { get; set; }
+
+        public Guid ShopId { get; set; }
```

Comment:
wtf? 
here you have a single shopId, but below list of shops. how is it possible?)